src/components/shared/SampleTooltipContents.js

```diff
@@ -139,10 +147,22 @@ export class SampleTooltipContents extends React.PureComponent<Props> {
         );
         hasStack = stack.length > 1 || stack[0].funcName !== '(root)';
       }
+
+      formattedSampleTime = getFormattedTimeLength(
```

Comment:
(optional) because it's a timestamp and not a time length, I find it a bit weird to call this function. I'd call `formatTimestamp` directly (the component `Ruler.js` does that).

Reply:
Technically it's the time length starting from the profiling start time. So I don't think it's too bad.